### ISSUE TITLE
Add the ability to change the OSC port directly from the control screen

### ISSUE THREAD

**Issue by AndrewRahman** [Spatial-Media-Lab/Lemur-OSC-Controller-for-iPad#4]

**Comment by aphleps** [comment]
I don't think this is possible. Lemur allows us to set up to 8 OSC targets (0-7) which are set in the preferences of the given device. The Lemur template addresses this targets by their ID (for example OSC target 4). This target may be a different one on another device.
